activerecord/test/cases/adapter_test.rb

```diff
@@ -36,6 +36,21 @@ def test_table_exists?
       assert !@connection.table_exists?(nil)
     end
 
+    def test_data_sources
+      tables = @connection.data_sources
```

Comment:
I feel `data_sources = @connection.data_sources` is better because name of this test is "test_data_sources"